src/kernels/raw/launcher/kernelProcess.node.ts

```diff
@@ -265,7 +265,7 @@ export class KernelProcess implements IKernelProcess {
         }
     }
 
-    public async dispose(): Promise<void> {
+    public dispose(): void {
```

Comment:
No async code here, hence no need to make this async.
The change results in connection files getting deleted when vscode is shutdown.

This could also mean that other parts of the code have similar issues and cleanup might not work as expected.